src/components/CardHeader.jsx

```diff
@@ -2,8 +2,13 @@ import timeSince from "./Utils";
 import style from "../styles/CardHeader.module.css";
 
 // CardHeader -> ~ago 표시되는 부분(card.createdAt 활용) + Utils.js의 timeSince 활용
-function CardHeader({ createdAt }) {
-  return <span className={style.card_header}>{timeSince(createdAt)}</span>;
+function CardHeader({ createdAt, showKebabMenu }) {
```

Comment:
P1:
showKebabMenu이 boolean 값인데 이름만 보면 함수처럼 보이네요. 이름으로 어떤 값인지 알 수 있게 해주시면 좋겠습니다.

```suggestion
function CardHeader({ createdAt, isShowKebabMenu }) {
```